config/lib/app.js

```diff
@@ -47,6 +47,7 @@ module.exports.start = function start(callback) {
       console.log(chalk.green('App version:     ' + config.meanjs.version));
       if (config.meanjs['meanjs-version'])
         console.log(chalk.green('MEAN.JS version: ' + config.meanjs['meanjs-version']));
+      console.log(chalk.green('Node version:    ' + process.version.substr(1)));
```

Comment:
I do prefer the location of this line in Mikael's PR. Seems more apropos for it to be grouped with the app, server, and db settings display.